bundles/org.openhab.binding.omnilink/src/main/java/org/openhab/binding/omnilink/internal/handler/OmniLinkCmd.java

```diff
@@ -0,0 +1,198 @@
+    CMD_SECURITY_LUMINA_HOME_MODE(49),
+    CMD_SECURITY_LUMINA_SLEEP_MODE(50),
+    CMD_SECURITY_LUMINA_AWAY_MODE(51),
+    CMD_SECURITY_LUMINA_VACATION_MODE(52),
+    CMD_SECURITY_LUMINA_PARTY_MODE(53),
+    CMD_SECURITY_LUMINA_SPECIAL_MODE(54),
+
+    CMD_BUTTON(7),
+
+    CMD_ENERGY_SAVER_OFF(64),
+    CMD_ENERGY_SAVER_ON(65),
+
+    CMD_THERMO_SET_HEAT_LOW_POINT(66),
+    CMD_THERMO_SET_COOL_HIGH_POINT(67),
+    CMD_THERMO_SET_SYSTEM_MODE(68),
+    CMD_THERMO_SET_FAN_MODE(69),
+    CMD_THERMO_SET_HOLD_MODE(70),
+    CMD_THERMO_RAISE_LOWER_HEAT(71),
+    CMD_THERMO_RAISE_LOWER_COOL(72),
+    CMD_THERMO_SET_HUMDIFY_POINT(73),
+    CMD_THERMO_SET_DEHUMIDIFY_POINT(74),
+
+    CMD_MESSAGE_SHOW_MESSAGE_WITH_BEEP_AND_LED(80),
+    CMD_MESSAGE_SHOW_MESSAGE_WITH_BEEP_OR_LED(86),
+    CMD_MESSAGE_LOG_MESSAGE(81),
+    CMD_MESSAGE_CLEAR_MESSAGE(82),
+    CMD_MESSAGE_SAY_MESSAGE(83),
+    CMD_MESSAGE_PHONE_AND_SAY_MESSAGE(84),
+    CMD_MESSAGE_SEND_MESSAGE_TO_SERIAL_PORT(85),
+
+    CMD_CONSOLE_ENABLE_DISABLE_BEEPER(102),
+    CMD_CONSOLE_BEEP(103),
+
+    CMD_LOCK_DOOR(105),
+    CMD_UNLOCK_DOOR(106),
+
+    CMD_AUDIO_ZONE_SET_ON_MUTE(112),
+    CMD_AUDIO_ZONE_SET_VOLUME(113),
+    CMD_AUDIO_ZONE_SET_SOURCE(114),
+    CMD_AUDIO_ZONE_SELECT_KEY(115),
+
+    SENSOR_UNIT_POWER(1001),
+    SENSOR_UNIT_LEVEL(1002),
+    SENSOR_UNIT_DISPLAY(1003),
+    SENSOR_THERMO_HEAT_POINTC(2001),
+    SENSOR_THERMO_HEAT_POINTF(2002),
+    SENSOR_THERMO_COOL_POINTC(2003),
+    SENSOR_THERMO_COOL_POINTF(2004),
+    SENSOR_THERMO_SYSTEM_MODE(2005),
+    SENSOR_THERMO_FAN_MODE(2006),
+    SENSOR_THERMO_HOLD_MODE(2007),
+    SENSOR_THERMO_TEMPC(2006),
+    SENSOR_THERMO_TEMPF(2007),
+    SENSOR_ZONE_STATUS_CURRENT(3001),
+    SENSOR_ZONE_STATUS_LATCHED(3002),
+    SENSOR_ZONE_STATUS_ARMING(3003),
+    SENSOR_AREA_STATUS_MODE(4001),
+    SENSOR_AREA_STATUS_ALARM(4002),
+    SENSOR_AREA_STATUS_EXIT_DELAY(4003),
+    SENSOR_AREA_STATUS_ENTRY_DELAY(4003),
+    SENSOR_AREA_EXIT_TIMER(4004),
+    SENSOR_AREA_ENTRY_TIMER(4005),
+    SENSOR_AUX_STATUS(5001),
+    SENSOR_AUX_CURRENTC(5002),
+    SENSOR_AUX_CURRENTF(5003),
+    SENSOR_AUX_LOWC(5004),
+    SENSOR_AUX_LOWF(5005),
+    SENSOR_AUX_HIGHC(5006),
+    SENSOR_AUX_HIGHF(5007),
+    SENSOR_AUDIOZONE_POWER(6001),
+    SENSOR_AUDIOZONE_SOURCE(6002),
+    SENSOR_AUDIOZONE_VOLUME(6003),
+    SENSOR_AUDIOZONE_MUTE(6004),
+    SENSOR_AUDIOZONE_TEXT(6005),
+    SENSOR_AUDIOZONE_TEXT_FIELD1(6006),
+    SENSOR_AUDIOZONE_TEXT_FIELD2(6007),
+    SENSOR_AUDIOZONE_TEXT_FIELD3(6008),
+    SENSOR_AUDIOSOURCE_TEXT(7001),
+    SENSOR_AUDIOSOURCE_TEXT_FIELD1(7002),
+    SENSOR_AUDIOSOURCE_TEXT_FIELD2(7003),
+    SENSOR_AUDIOSOURCE_TEXT_FIELD3(7004);
+
+    private int number;
+
+    OmniLinkCmd(int number) {
+        this.number = number;
+    }
+
+    public int getNumber() {
+        return number;
+    }
+
+    public static @Nullable OmniLinkCmd getCommand(String name) {
+        for (OmniLinkCmd command : OmniLinkCmd.values()) {
```

Comment:
Why don't you use an `Optional` here? ou do that in other enums.

Reply:
I did not write this code, I simply left it how it was as it was still working just fine. I try to operate on the principle of don't fix what isn't broken.